containers/deploy/pulp/deployments.yml

```diff
@@ -91,9 +91,7 @@
                             - readOnly: false
                               mountPath: /etc/puppet
                               name: puppet-data
-                          image: projgriffin/foreman-pulp-worker:latest
-                      serviceAccount: anyuid
-                      serviceAccountName: anyuid
```

Comment:
Wouldn't this still need anyuid because you specified `user: apache` above for this container?

Reply:
Ah, I see this is the stop

Reply:
So, I did drop this here -- https://github.com/ehelms/forklift/pull/6/files#diff-57ebeb493820adb1bcdc8470d1d43cd4L384

When I ran this without it, the pulp worker functioned as I expected it to making this seem unneccessary? Or is there more to dropping this that I am missing?

Reply:
Were you running with the same cluster_up playbook? Because then it would have anyuid either way because everyone is part of the `system:authenticated` group.

Looking at https://docs.openshift.org/latest/rest_api/oapi/v1.DeploymentConfig.html it seems like we only need `serviceAccountName` (`serviceAccount` is deprecated). But I don't see it in the container spec like you have [here](https://github.com/ehelms/forklift/pull/6/files#diff-57ebeb493820adb1bcdc8470d1d43cd4R370). Would think that would cause an error :confused:

Reply:
I updated to put this back for now. Note I also updated the cluster-up playbook to drop the account anyuid setting.